site/src/content/docs/forms/form-control.mdx

```diff
@@ -1,58 +1,49 @@
 ---
-layout: docs
 title: Form controls
-description: Give textual form controls like `<input>`s and `<textarea>`s an upgrade with custom styles, sizing, focus states, and more.
```

Comment:
Astro's doesn't seem to support `<input>` in the description as it doesn't transform it to `<code><input></code>`